package/network/utils/comgt/files/ncm.sh

```diff
@@ -150,6 +164,14 @@ proto_ncm_setup() {
 		proto_add_dynamic_defaults
 		ubus call network add_dynamic "$(json_dump)"
 	}
+
+	echo "finalize connection"
```

Comment:
Do you think these extra echos are useful? I think runcommand.gcom is already noisy enough about the commands it is sending... If you decide to leave them, change this to "Finalizing connection" for consistent capitalization.